### PR TITLE
FIX: 0-index ICA channel names in get_sources

### PR DESCRIPTION
**Pull request by kdoelling1919**
Tiny fix to make ICA channel names 0-indexed in the output of ica.get_sources as they are in the ICA plot functions.

@dengemann

### CODE REVIEW

File: mne/preprocessing/ica.py [hunk 1]
```diff
@@ -786,7 +786,7 @@ def _export_info(self, info, container, add_channels):
         ch_names = []
         ch_info = info['chs'] = []
         for ii in range(self.n_components_):
-            this_source = 'ICA %03d' % (ii + 1)
+            this_source = 'ICA %03d' % (ii)
```
Comment (agramfort) [inline mne/preprocessing/ica.py]: channel names for ICA are called ICA 001, ICA 002 etc.

if you want to display as number it should be displayed as ICA #0, ICA #1 etc.

Comment (mmagnuski) [inline mne/preprocessing/ica.py]: I also like `IC #000`, but you should make sure that the naming is consistent across mne.
BTW @agramfort managed to reference first PR to mne python :)

Comment (kdoelling1919) [inline mne/preprocessing/ica.py]: @mmagnuski You mean that we should ensure consistency across mne with regards to IC plots, yes? Not that we should 0-index the naming for all channel types. I am a fan of 0-indexing everything but I guess other channel names are one-based indexed for a reason.

A great accidental reference from @agramfort

**Comment by agramfort** [comment]
we discussed this many times

conclusion so far is:
- name ICs as `IC #0`, `IC #1`, etc. in plots
- name channels in ch_names and info['chs'] as `ICA 001` `ICA 002` etc. for consistency with EEG 001, EEG 002 etc.

**Comment by kdoelling1919** [comment]
Okay fair enough. Thanks for clearing it up. 

Actually, as I was trying to figure out evoked.plot functionality regarding this I couldn't actually get it to plot without error (before all this, I was just extracting the data and using plt.plot()). The error I found is described in Issue #3647 

Perhaps, after that is sorted out, we can come back to this. I'm wondering what you think about creating an option for the user to change format of plot legends. Perhaps (probably) that is already implemented and I don't know about it.

**Comment by agramfort** [comment]
closing?

**Pull request closed without merging**